docs/webserver-endpoints.md

```
@@ -204,40 +204,41 @@ Folder created: NewFolder
 
-| Parameter | Required | Default | Description                      |
-| --------- | -------- | ------- | -------------------------------- |
-| `path`    | Yes      | -       | Path to the item to delete       |
-| `type`    | No       | `file`  | Type of item: `file` or `folder` |
+| Parameter | Required | Default | Description |
+| --------- | -------- | ------- | ----------- |
+| `path`    | Yes, unless `paths` is provided | - | Path to one item to delete |
+| `paths`   | Yes, unless `path` is provided | - | JSON array of paths to delete |
 
 **Response (200 OK):**
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)